packages/schema/src/types/hooks.ts

```diff
@@ -73,6 +73,11 @@ export interface NuxtHooks {
    * @returns Promise
    */
   'close': (nuxt: Nuxt) => HookResult
+  /**
+   * Called when Nuxt instance is restarting.
```

Comment:
Would you also add this to the docs? https://github.com/nuxt/nuxt/blob/b25b6c82c7fbd6dc3f3378c4bcac8bcc5ce361e5/docs/3.api/4.advanced/1.hooks.md#L32